get-started/versioning/index.json

```diff
@@ -6,27 +6,27 @@
     "details": {
         "steps": [
             {
-                "title": "Track a file or directory",
+                "title": "Step 1",
                 "text": "01-track-a-file-or-directory.md"
```

Comment:
Alternatively we could keep the full titles here and remove the `#` (H1) titles from .md files.

Reply:
I guess that how it was before this change. OK, as you prefer 👍